### DISCUSSION TITLE
Missing pages in TIFF multipage

### DISCUSSION THREAD

**Discussion by KFoster2** [ImageMagick/ImageMagick#2036] [Help]
Hello,

I´m using this code with a TIFF multipage but only get the first page, not the other ones:

convert sample.tiff ( -clone 0 -blur 0x5 ) +swap -compose divide -composite output.tiff

Am I missing anything in my code?

Thanks!

**Comment by dlemstra** [comment]
Could you attach the `sample.tiff` file?

**Comment by fmw42** [comment]
What are you trying to do? Your -clone 0 is only getting the first page of the input TIFF file, since the clone refers to the images in the command sequence. ImageMagick sees the TIFF as multiple images, not just one. 

I suspect you want to divide each page into its blurred version. That would require -layers composite and cloning every page as follows

`convert sample.tiff null: ( -clone 0--2 -blur 0x5 ) -reverse -compose divide -layers composite -reverse output.tif`

The 0--2 clones all images in the input sequence from the first to the second from the last. That is because we do not want to clone the null: image separator.


See 
https://imagemagick.org/Usage/basics/#clone
https://imagemagick.org/Usage/layers/#layer_composite
https://imagemagick.org/Usage/anim_mods/#composite

**Comment by KFoster2** [comment]
Thanks fmw42. You´re absolutely right, I was not using the code properly as I was getting the first page only. It works ok now!

**Comment by ghacky** [comment]
Kind of a hijack, but i'm trying to put multiple jpg's into a single pdf, one jpg per page.  I used magick *.jpg out.pdf
but i only get the first jpg, the rest are blank, though the output file size seems to be the sum of the individual sizes.  Any ideas?

**Reply by snibgo** [reply]
Your question is entirely different, so please start a new thread.
